### DISCUSSION TITLE
port/live555/usage not found

### DISCUSSION THREAD

**Discussion by knzsoft** [microsoft/vcpkg#28153] [Q&A]
How to find live555 in CMake?

**Comment by JackBoosY** [comment]
live555 doesn't export its targets: https://github.com/microsoft/vcpkg/blob/master/ports/live555/CMakeLists.txt
I will make a PR to add it later.

**Reply by JackBoosY** [reply]
https://github.com/microsoft/vcpkg/pull/28252

**Reply by JackBoosY** [reply]
Please git pull vcpkg and rebuild live555.
